scripts/fast-reboot

```diff
@@ -21,6 +21,8 @@ PLATFORM_PLUGIN="${REBOOT_TYPE}_plugin"
 LOG_SSD_HEALTH="/usr/local/bin/log_ssd_health"
 SSD_FW_UPDATE="ssd-fw-upgrade"
 TAG_LATEST=yes
+DETACH=yes
```

Comment:
I think -d should mean detach and default should be DETACH=no

Reply:
Sorry for late response. The idea of the changes is to prevent failures of warm-reboot/fast-reboot in cases when the ssh session is broken/closed. I made detached mode as default and leave possibility to run the scripts in non-detached mode with -d if the user wants. If to make as you propose, the user could forget to add -d parameter, what may lead to failure of the reboot. Due to this, I prefer to leave parameter -d as is. By the way, parameter -t made in similar way, so, I thought that there is no problem to let -d to run the process in non-detached mode, not wise versa.

Reply:
@maksymbelei95 I don't think changing default behavior is the right thing. This blocking behavior has been used until now in tests and production. Changing the default behavior could have unexpected consequences. For special scenario, it should not be default.